extensions/applicationinsights-debugplugin-js/src/components/styleNodeSrc.ts

```diff
@@ -0,0 +1,139 @@
+export const tempStyle = `
```

Comment:
Any reason this is not in a CSS file?

Reply:
these two style variables are dynamically injected at runtime when the plugin is initialized.
there are a number of things we could do with this. One such thing is opening the door for potentially user-configurable styles in the future as well as allowing me to dynamically add and remove the styles as necessary to minimize styling conflicts.

Reply:
same *possible* name conflict issue -- prefix with at least "ai-", that way we have at least a story about grouping all app insights styling etc.